src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -67,6 +73,10 @@ public static HearingDetails buildHearingDetails(HearingWrapper wrapper, Referen
             .build();
     }
 
+    public static boolean isHearingAdjourned(SscsCaseData caseData) {
+        return isNotBlank(caseData.getAdjournCaseGenerateNotice());
```

Comment:
Checking really but would the adjournCaseGenerateNotice only be present if a hearing is adjourned? Does this reset if the case is no longer in an adjourned state? In other words, will this correctly identify is a hearing is adjourned more than once? (Granted I'm out of adjournment development completely so not super up to speed on the intricacies, correct me if I'm off base).

Reply:
Unsure about this, will check

Reply:
I have changed this now to check if adjournCaseGeneratedDate == today. This is only a temporary solution which should work fine and suffice for this specific ticket. 
The long-term solution, and one which will be used by all adjournment tickets, is to implement a new field on SscsCaseData called isAdjournmentInProgress. 
It will be treated much like isUnprocessedPostponement in the method above. ie It will be set to YES at the point of adjournment, then toggled to NO in hearings by HearingsServiceHelper#clearTransientFields.
Implementing this will be my next immediate task, and should be attached to its own ticket.